src/screens/AddParticipant/index.js

```diff
@@ -262,6 +262,6 @@ const styles = StyleSheet.create({
     marginTop: 16
   },
   subtitleStyle: (isIsset) => ({
-    color: isIsset ? COLORS.anon_primary : COLORS.lightgrey
+    color: isIsset ? COLORS.anon_primary : COLORS.gray100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.lightgrey` to `COLORS.gray100` in the subtitleStyle could affect visibility and contrast depending on the background. Please ensure that this change doesn't negatively impact the user experience.
```diff
-    color: isIsset ? COLORS.anon_primary : COLORS.lightgrey
+    color: isIsset ? COLORS.anon_primary : COLORS.gray100
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->